src/tasks/dump.ts

```diff
@@ -492,8 +492,8 @@ export async function dump({
       network,
       api,
     });
-  if (instructions.length === 0) {
-    console.log("No token can be sold");
+  if (instructions.length === 0 && transferToReceiver === undefined) {
+    console.log("Nothing to do");
```

Comment:
I can imagine there is "nothing to do" in both cases, but doesn't the second one imply a bit more... like a failure to determine receiver? I can imagine that running the script as `--dryRun` we might choose not to set this `transferToReceiver`, but is that the only scenario?

Reply:
I changed how logs are printed and conditions are checked. Does it help?